### PR TITLE
[MB-2053] Release 6.7.0 of the Urban Airship Cordova plugin

### PR DESCRIPTION
**Pull request by BrianBatchelder**
* iOS build now requires a build.json to enable code signing.
 * iOS build now has a “before_prepare” hook that changes the pod resource build script to look for the resource bundle in two places: one used by the Xcode build, the other by the Cordova build.
 * ‘Alias’ functionality is now deprecated in the plugin. It will be removed in a future version of the plugin. Please use 'Named User'.

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -176,6 +179,6 @@
         <header-file src="src/ios/UAMessageViewController.h" />
         <source-file src="src/ios/UAMessageViewController.m" />
 
-        <framework src="UrbanAirship-iOS-SDK" type="podspec" spec="~> 8.0.5" />
+        <framework src="UrbanAirship-iOS-SDK" type="podspec" spec="~> 8.3.0" />
```
Comment (rlepinski) [inline plugin.xml]: 8.3.1

File: src/android/build-extras.gradle [hunk 2]
```diff
@@ -11,12 +11,15 @@ android {
...
-    compile 'com.android.support:support-annotations:25.1.0'
-    compile 'com.google.android.gms:play-services-gcm:[9.4.0,)'
-    compile 'com.google.android.gms:play-services-location:[9.4.0,)'
-    compile 'com.urbanairship.android:urbanairship-sdk:8.2.5'
+    compile 'com.android.support:support-v4:25.3.1'
+    compile 'com.android.support:support-annotations:25.3.1'
+    compile 'com.google.android.gms:play-services-gcm:10.2.1'
+
+    compile 'com.android.support:cardview-v7:25.3.1'
+    compile 'com.google.android.gms:play-services-location:10.2.1'
```
Comment (rlepinski) [inline src/android/build-extras.gradle]: [10.2.1,)

File: src/android/build-extras.gradle [hunk 3]
```diff
@@ -11,12 +11,15 @@ android {
...
 dependencies {
-    compile 'com.android.support:support-v4:25.1.0'
-    compile 'com.android.support:cardview-v7:25.1.0'
-    compile 'com.android.support:support-annotations:25.1.0'
-    compile 'com.google.android.gms:play-services-gcm:[9.4.0,)'
-    compile 'com.google.android.gms:play-services-location:[9.4.0,)'
-    compile 'com.urbanairship.android:urbanairship-sdk:8.2.5'
+    compile 'com.android.support:support-v4:25.3.1'
+    compile 'com.android.support:support-annotations:25.3.1'
+    compile 'com.google.android.gms:play-services-gcm:10.2.1'
```
Comment (rlepinski) [inline src/android/build-extras.gradle]: [10.2.1,)
It will force min version of 10.2.1 but allow newer versions

Comment (rlepinski) [inline src/android/build-extras.gradle]: http://ant.apache.org/ivy/history/2.1.0/settings/version-matchers.html

File: plugin.xml (hunk 1 above)
Comment (BrianBatchelder) [inline plugin.xml]: How did I miss that? ;-)

**Comment by rlepinski** [comment]
👍

**Comment by BrianBatchelder** [comment]
Release 6.7.0
 - Updated iOS Urban Airship SDK to 8.3.2
 - Updated Android Urban Airship SDK to 8.4.1
 - ‘Alias’ functionality is now deprecated in the plugin. It will be removed in a future version of the plugin. Please use 'Named User'.
 - iOS: Use static library until dynamic framework build issues with Cordova and Cocoapods have been resolved.
 - Android: fix landing page issues

**Pull request merged**